Snakefile

```diff
@@ -70,6 +73,15 @@ rule prepare_feedin:
     shell:
         "python {input.script} {input.wind_feedin} {input.pv_feedin} {output}"
 
+rule prepare_electricity_demand:
+    input:
+        opsd_url=HTTP.remote("https://data.open-power-system-data.org/time_series/2020-10-06/time_series_60min_singleindex.csv"),
+        script="scripts/prepare_electricity_demand.py"
+    output:
+        "results/_resources/load_profile_time_series.csv"
```

Comment:
```suggestion
        "results/_resources/load_profile_electricity.csv"
```

Reply:
There will be other load profiles which have to reside in different files to have unique outputs in snakemake, therefore this name should be specific.